lib/js/jslib.js

```diff
@@ -2922,6 +2923,20 @@ function spawnAt(loc, f, kappa) {
   return _applyCont(kappa,_spawnAt(loc, f));
 }
 
+function _newAP(loc) {
+    check_location(loc);
+    return _new();
+}
+
+function _newClientAP() { return _new(); }
+function _newServerAP() {
+    throw new Error('Cannot create a server access point from a client');
+}
+
+const newClientAP = LINKS.kify(_newClientAP);
```

Comment:
Is there a good reason why `const new = LINKS.kify(_new);` doesn't appear somewhere in `jslib.js`?

Reply:
I wouldn't be surprised if it's because `new` is a keyword.

Reply:
That's probably it... but we should really have a way with dealing with that kind of name clash. Something to think once #1087 gets merged.